docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md

```diff
@@ -110,3 +110,12 @@ speech_sample -i feats.ark,ivector_online_ie.ark -m final.xml -d CPU -o predicti
 
 Results can be decoded as described in "Use of Sample in Kaldi* Speech Recognition Pipeline" chapter 
 in [the Speech Recognition Sample description](../../../../../inference-engine/samples/speech_sample/README.md).
+
+---
+## See Also
+
+* [Converting a Style Transfer Model from MXNet*](../mxnet_specific/Convert_Style_Transfer_From_MXNet.md)
```

Comment:
Remove all references. May be add a reference to Convert models from Kaldi only.